programs/fileio.c

```diff
@@ -1130,13 +1143,15 @@ FIO_compressLz4Frame(cRess_t* ress,
     LZ4F_preferences_t prefs;
     LZ4F_compressionContext_t ctx;
 
+    IOJob_t *writeJob =AIO_WritePool_acquireJob(ress->writeCtx);
```

Comment:
minor (coding style): 
if `writeJob` pointer never changes after initialization, you should `const` it : `IOJob_t* const writeJob = ...`.
It sends a signal to the reviewer (or the next contributor) that this pointer will never change its value later on.
When applied consistently, its _absence_ becomes a signal that this pointer _will_ change in the future, and should therefore receive extra attention.

Reply:
It actually does change later on with 
```
AIO_WritePool_enqueueAndReacquireWriteJob(&writeJob);
```